src/components/UserInfo.js

```diff
@@ -11,8 +18,15 @@ export default class UserInfo {
     };
   }
 
-  setUserInfo({ name, info }) {
+  setUserImage(avatar) {
+    this._userImage.src = avatar;
+  }
```

Comment:
`setUserImage` duplicates the code in line 28. But you don't need it anymore because you need to use only `setUserInfo` everywhere in the code

Reply:
I use the setUserImage method on line 159 of index.js to update the _userImage property when the user updates the profile image using that pop-up. Since only the image is changing here, I thought it was appropriate to have a separate method.

Reply:
> I use the setUserImage method on line 159 of index.js to update the _userImage property when the user updates the profile image using that pop-up. Since only the image is changing here, I thought it was appropriate to have a separate method.

 The thing is that you duplicate the code and duplication of code is not allowed by the checklist


So, you just need to use only `setUserInfo` (like you do in line 141 when you edit the profile data (you set all 4 params including the avatar there)


```js

      .then((res) => {
        userInfo.setUserInfo(res);
```

Reply:
Ok, I understand. Thank you for the explanation.